### PR TITLE
Added new annotations from my notes

### PR DESCRIPTION
**Pull request by abrarhayat**

### CODE REVIEW

File: features.md
Comment (travisddavies) [inline features.md:1]: Looks ok to me, just add \text{ or } in latex instead so it doesn't look like a math term and I think it's all good

**Review by travisddavies — COMMENTED** [review]
This change is actually incorrect, the additional of alpha and beta equals the disparity. It's also this answer in the solution

File: features.md
Comment (abrarhayat) [reply features.md:1]: Fixed

Comment (abrarhayat) [reply features.md:1]: @travisddavies I thought alpha and beta were x and x' Which are the corresponding coordinates of the right and left images in the stereo setup, but I am guessing alpha and beta were different values?

Comment (travisddavies) [reply features.md:1]: @abrarhayat  Nah in this case alpha and beta aren't locations, they're purely distances in the x direction from the point. So combining them gives you x - x'

Comment (abrarhayat) [reply features.md:1]: I changed it back in the latest commit.

Comment (travisddavies) [reply features.md:1]: Sweet

**Review by travisddavies — APPROVED** [review]

**Pull request merged**